notebooks/exploring_explorer.livemd

```diff
@@ -336,6 +332,24 @@ Or the tail. Let's get a few more values from the tail.
 DataFrame.tail(df, 10)
 ```
 
+### Verbs and macros
+
+In `Explorer`, like in `dplyr`, we have five main verbs to work with dataframes:
+
+* select
+* filter
+* mutate
+* arrange
+* summarise
+
+We are going to explore then in this notebook, but first we need to "require"
+the `Explorer.DataFrame` module in order to load the macros needed for these verbs.
+Since we have an alias for that module, we just need to:
+
+```elixir
+require DataFrame
```

Comment:
Let's change `alias Explorer.DataFrame` at the top to `require Explorer.DataFrame, as: DF`. WDYT?

Reply:
However, if we do so, should we alias `Explorer.Series` to `S`? Or maybe we start with `alias Explorer.Series` only and introduce the `require Explorer.DataFrame, as: DF` later?

Reply:
I changed to introduce the `DF` alias here (the second option). WDYT?